core/src/main/java/software/amazon/awssdk/core/sync/StreamingResponseHandler.java

```diff
@@ -104,13 +106,25 @@ default boolean needsConnectionLeftOpen() {
      * @param <ResponseT>  Type of unmarshalled response POJO.
      * @return StreamingResponseHandler instance.
      */
+    @ReviewBeforeRelease("How does this know not to retry writing to the output stream if the copy fails?")
```

Comment:
By not throwing a RetryableException.

Reply:
I didn't see any of them throwing a retryable exception. Does it need to be fixed in the path one as well, or did I miss it?

Reply:
Yeah if they aren't throwing it then they won't be retried.

Reply:
I'll add it to those as well as this one.